x/cron/keeper/keeper.go

```diff
@@ -205,6 +209,7 @@ func (k *Keeper) executeSchedule(ctx sdk.Context, schedule types.Schedule) error
 			)
 			return err
 		}
+		defer telemetry.ModuleMeasureSince(types.ModuleName, startTimeContract, LabelExecuteCronContract, schedule.Name, msg.Contract)
```

Comment:
Hm, suppose we have two schedules:A and B. A gets called first, B is second.
Doesn't this `defer` call mean, that a measurement for A call contains a time consumption of the B call?
Because the `ModuleMeasureSince` is called only in the end of the whole execution.

Reply:
Resloved